### ISSUE TITLE
some problems to install rable on windows

### ISSUE THREAD

**Issue by shakhanabdiev** [nesquena/rabl#740]
`gem install rabl`

C:\Users\User\Documents\Projects\e_commerce>gem install rabl
ERROR:  While executing gem ... (Errno::EACCES)
    Permission denied @ rb_file_s_symlink - (fixtures/ashared/views_rails_3/users/phone_number.json.rabl, C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/rabl-0.14.3/fixtures/ashared/views_rails_3/users/phone_number.xml.rabl)

**Comment by tagliala** [comment]
Hi,

thanks for reporting this

I can confirm the issue. This is caused by symlinks that do not work properly on windows

I do not use windows as a development machine and I can't clearly find a solution on google.

Would rather duplicate the files and 👋 👋

**Comment by Ex-Ark** [comment]
Same issue, symlink not working on windows

Workaround is to build from source  :
```
git clone https://github.com/nesquena/rabl
cd rabl
# optional
# git checkout whatever version you need
gem build rabl.gemspec
```

should output 
```
 Successfully built RubyGem
  Name: rabl
  Version: 0.14.3
  File: rabl-0.14.3.gem
```

Then you can install it system wide, so it should be properly picked up by any project (Gemfile refering to this specific version).
```
gem install rable-0.14.3.gem
```

Proper & prod ready solution would be for the lib to drop the use of symlink and hard copy the file itself.
Especially if it's fixtures data that's crashing the gem install on other OS.

for my case it was `phone_number.json.rabl` as well

**Comment by tagliala** [comment]
I'm pretty confident that this has been fixed in v0.14.4 via ea31f21cadb9df0490914dd991bca4c3a904a7a5
